.github/workflows/label.yml

```diff
@@ -3,14 +3,15 @@ on:
   pull_request_target:
     types: closed
 
+permissions:
+  contents: read
+  
 jobs:
   label:
 
     runs-on: ubuntu-latest
     permissions:
-      contents: read
-      issues: write
-
+      pull-requests: write # Required to add labels to pull-requests
```

Comment:
It's  not adding labels to pull-requests, but rather issues. So guess should be issues: write.

Reply:
@stianst maybe I completely misunderstood it but the following line: https://github.com/keycloak/keycloak/blob/main/.github/workflows/label.yml#L2-L4, pretty much states that workflow runs when a pull request (PR) is closed (either merged or explicitly closed without merging). That's the reason why `pull-requests: write` was added instead of `issues`.

Reply:
It doesn't need permissions to run on the event, it needs permissions for what actions it takes, which is update issues and create labels

Reply:
+1 this seems like it could use more granular permissions